src/pages/LandingPages/MGCovid19/CovidLandingForm.vue

```diff
@@ -163,8 +176,43 @@ export default {
 			onetimeAmount: 50,
 			minOnetimeAmount: 25,
 			maxDepositAmount: 10000,
+			isExperimentActive: false
 		};
 	},
+	inject: ['apollo'],
+	apollo: {
+		query: pageQuery,
+		preFetch(config, client) {
+			return client.query({
+				query: pageQuery
+			}).then(() => {
+				return client.query({
+					query: experimentQuery, variables: { id: 'covid19response_default_amount' }
+				});
+			});
+		},
+		result() {
+			// GROW-96 Experiment
+			const covid19responseDefaultAmount = this.apollo.readFragment({
+				id: 'Experiment:covid19response_default_amount',
+				fragment: experimentVersionFragment,
+			}) || {};
+			this.isExperimentActive = covid19responseDefaultAmount.version === 'shown';
+			if (this.isExperimentActive) {
+				this.recurringAmountSelection = '25';
+				this.recurringCustomAmount = 25;
+				this.recurringAmount = 25;
+			}
+		},
+	},
+	mounted() {
+		// Fire Event for GROW-96
+		this.$kvTrackEvent(
```

Comment:
@mlibazisi Overall looks good. For this analytics call we should wrap it in a check that looks at the experiment setting to see if it is enabled or not. We don't want to fire this event if we haven't yet started the experiment.

This update will allow us to deploy the code then turn it on at a later date.

Reply:
@mcstover Should I set it as a separate admin setting that i will need to query for?

Reply:
It will be present on the result of the `pageQuery` variable above so you could do a readQuery with query set to `pageQuery` and get the full experiment setting.